frontend/pages/index.tsx

```diff
@@ -20,7 +20,10 @@ export default ({ posts }) => (
             Recent Community Blog Posts
           </h2>
           <p className="mt-3 max-w-2xl mx-auto text-xl leading-7 text-gray-500 sm:mt-4">
-            What's been talked about in the CDK ecosytem
+            What's been talked about in the CDK ecosytem.{" "}
```

Comment:
What's the `{" "}` for?

Reply:
No intentionally there, equals to a space. Will remove it as well